adapters/oidc/adapter-core/pom.xml

```diff
@@ -73,6 +73,10 @@
             <artifactId>keycloak-core</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>${keycloak.crypto.artifactId}</artifactId>
```

Comment:
Shouldn't this just be `keycloak-crypto-default`? We shouldn't put any effort into FIPS around adapters since they are being deprecated.

Reply:
Agree that we shouldn't put effort into FIPS in adapters. And in fact the changes in the adapter WF modules in this PR have it hardcoded.

However there are some other potential things that for example our testsuite has dependency on `keycloak-adapter-core`. In case it is hardcoded here to `keycloak-crypto-default`, then when the testsuite is executed with the `fips` profile, it will bring to the classpath both the `keycloak-crypto-fips` (through keycloak-services) and the `keycloak-crypto-default` (through keycloak-adapter-core), making it probably some issues due the conflicts due both BC and BC-FIPS would be on the classpath.